noxfile.py

```diff
@@ -75,7 +75,7 @@ def _session_tests(session, folder):
 @nox.parametrize("folder", FOLDERS)
 def py(session, folder):
     """Runs py.test for a folder using the specified version of Python."""
-    session.install("-r", "test-requirements.txt")
+    session.install("-r", "requirements-test.txt")
```

Comment:
This was originally referring to this file: https://github.com/GoogleCloudPlatform/fourkeys/blob/main/test-requirements.txt

We should probably put all the pytest installments in this root level file so we don't have to keep installing the same dependencies on all the sub-directories.  WDYT?

Reply:
Hmm! That might make the CI tests faster(?). At the risk of overcomplicating, maybe we do a global `requirements-test`, which then references all the individual ones. I’ll open an issue to explore this.